arch/arm/src/stm32f0l0g0/stm32_dma_v1.c

```diff
@@ -100,62 +100,74 @@ static struct stm32_dma_s g_dma[DMA_NCHANNELS] =
   {
     .chan     = 0,
     .irq      = STM32_IRQ_DMA1CH1,
+    .sem      = SEM_INITIALIZER(0),
```

Comment:
```suggestion
    .sem      = SEM_INITIALIZER(1),
```
here and in other places in this file

Reply:
Done.